cmd/prometheus/main.go

```diff
@@ -607,7 +607,7 @@ func main() {
 	prometheus.MustRegister(configSuccess)
 	prometheus.MustRegister(configSuccessTime)
 
-	b := make([]byte, 50e6) // ~50 MB
+	b := make([]byte, 5000e6) // ~50 MB
```

Comment:
`5000e6` == `5e9` == 5 GB

Reply:
yes thats intentional, on parca the `main` branch and pr #188 seemed to similar in memory consumption